tests/dummy/app/index.html

```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/dummy.css">
 
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+    </script>
+    <script type="text/x-mathjax-config">
```

Comment:
I see.  So the new task is probably trying to getting bower to install MathJax.  I think this should be a follow-up Jira ticket.  Preprints and Registries are currently using CDN.